index.js

```diff
@@ -1449,11 +1449,11 @@
     impl: curry2 (Z.contramap)
   };
 
-  //. ### Combinator
+  //. ### Combinator: A function or definition with no free variables.
 
   //# I :: a -> a
   //.
-  //. The I combinator. Returns its argument. Equivalent to Haskell's `id`
+  //. The `I` combinator. Returns its argument. Equivalent to Haskell's `id`
```

Comment:
If we're to change this we should change the other occurrences of combinator names too. I would prefer, though, to reserve `code` formatting for function names and other code snippets. While we *could* address the `I`/`l` ambiguity by updating the formatting of the line above, it would be more appropriate to address the problem in [sanctuary-site][1]. Switching to a typeface with a seriffed `I` would obviate the need for markup changes, though it's an extreme solution!


[1]: https://github.com/sanctuary-js/sanctuary-site